script/Arcueid/Day 2/02_00_ARC02_4B.txt

```diff
@@ -274,7 +274,7 @@ It's hard to get a read on her because she doesn't really let her emotions show,
 [sha:4db8df14976c8fe2a2bee6f3dd4a120d471e9e85]{
 -- Page 217, Offset 3179.
 -- 翡翠ちゃんはああ見えてとっても優しいんですから。
-Hisui-chan is very kind in spite of her appearance.
+Despite how she seems, Hisui-chan is very kind.
```

Comment:
Not sure I love `how she seems`. Maybe go more interpretive 
> It may not be obvious at first glance, but Hisui...
> She may not look it, but Hisui...

Reply:
This conversation was marked resolved but no change was made. Did you talk offline?

Reply:
i just forgot to push the commit but i already had the file edited on my end